### DISCUSSION TITLE
Rotated thin section for additional data collection

### DISCUSSION THREAD

**Discussion by hannahcunning** [mtex-toolbox/mtex#1549] [Ask Anything] [answered]
Hello! 

I had to rotate my thin section to be able to collect data on the other edge and I want to make sure that I am not messing things up by dealing with the rotation incorrectly. I was unable to do this rotation before collecting so I since that choice did not appear so I used the same sample coordinate system as I did for my previous data collection. 

For all of my other ebsd data I have been plotting: 
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','intoPlane');

and for the new data I plotted it as: 
setMTEXpref('xAxisDirection','west');
setMTEXpref('zAxisDirection','intoPlane');

Is that all I need to do, or do I need to be doing the rotation with the Euler angles? 

I have included images to show how I rotated the thin section, and the rotated map and pole figure.
![TS orientation](https://user-images.githubusercontent.com/54119012/189684113-7497fa5a-23b1-4263-a7af-d3ae10a68502.jpg)

![TS rot](https://user-images.githubusercontent.com/54119012/189684129-d4cfb385-8d3a-4833-a820-913a3778caa3.jpg)

Thank you.

**Comment by zmichels** [comment, marked as answer]
Instead of changing the default plotting preferences between datasets, I would rotate the data after import and keep the plotting conversions the same for everything.

Based on the schematic, I would try something like:
ebsd=rotate(ebsd,rotation(‘axis’,zvector,’angle’,180*degree))

**Reply by hannahcunning** [reply]
Thank you! Works perfectly!
